specification/compute/resource-manager/Microsoft.Compute/stable/2022-04-04/CloudServiceRP/cloudService.json

```diff
@@ -2141,11 +2175,11 @@
         },
         "settings": {
           "description": "Public settings for the extension. For JSON extensions, this is the JSON settings for the extension. For XML Extension (like RDP), this is the XML setting for the extension.",
-          "type": "string"
+          "type": "object"
```

Comment:
just want to confirm, this is the bug we are addressing, right? 
we are going from string to object?

Reply:
Yes, that's correct.